### PR TITLE
DEVDOCS-5166: [update] Pricing, Get Prices Batch, include tax_discount_amount array to bath response

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-5166]

## What changed?

*  Add tax_discount_amount array to Get Prices (Batch) endpoint response
*  Apply style guide

## Anything else?

ping {names}

[DEVDOCS-5166]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5166?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/pricing.sf.yml [hunk 1]
```diff
@@ -45,39 +45,43 @@ paths:
               properties:
                 channel_id:
                   type: integer
-                  description: The channel context that pricing should be evaluated within. The default BC storefront is channel 1
+                  description: The channel ID that pricing evaluates within. The default BigCommerce storefront is channel 1.
                   example: 1
                 currency_code:
                   type: string
-                  description: The currency of prices to be returned for this request
+                  description: The currency code of prices for this return request.
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:52]:
```suggestion
                  description: The currency code of prices this request returns.

```

File: reference/pricing.sf.yml [hunk 2]
```diff
@@ -45,39 +45,43 @@ paths:
...
                   example: USD
                 customer_group_id:
                   type: integer
-                  description: 'The customer group relevant for any customer group pricing, tax values, etc.'
+                  description: The customer group ID that's relevant for any customer group pricing, tax values, etc.
                 items:
                   type: array
-                  description: The items to fetch prices for
+                  description: The items to fetch prices for.
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:59]:
```suggestion
                  description: The items for which to fetch prices.

```

File: reference/pricing.sf.yml [hunk 3]
```diff
@@ -117,81 +121,81 @@ paths:
...
-                              description: The estimated tax inclusive price for this product based on the provided customer group
+                              description: The estimated tax inclusive price for this product based on the provided customer group.
                           description: The (optional) RRP/retail price configured for this product. Used for price comparison and storefront display purposes.
                         sale_price:
                           type: object
                           properties:
                             as_entered:
                               type: number
-                              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:140]:
```suggestion
                              description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.
```

File: reference/pricing.sf.yml [hunk 4]
```diff
@@ -117,81 +121,81 @@ paths:
...
-                              description: The estimated tax inclusive price for this product based on the provided customer group
+                              description: The estimated tax inclusive price for this product based on the provided customer group.
                           description: The merchant-entered sale price for a product overwrites the default price. The sale_price is optional.
                         minimum_advertised_price:
                           type: object
                           properties:
                             as_entered:
                               type: number
-                              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:156]:
```suggestion
                              description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 5]
```diff
@@ -117,81 +121,81 @@ paths:
...
-                              description: The estimated tax inclusive price for this product based on the provided customer group
+                              description: The estimated tax inclusive price for this product based on the provided customer group.
                           description: The minimum advertised price (MAP) allowed to be shown on a storefront. A value supplied by the merchant and used for display purposes.
                         price:
                           type: object
                           properties:
                             as_entered:
                               type: number
-                              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:172]:
```suggestion
                              description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.
```

File: reference/pricing.sf.yml [hunk 6]
```diff
@@ -200,33 +204,33 @@ paths:
                               properties:
                                 as_entered:
                                   type: number
-                                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:207]:
```suggestion
                                  description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 7]
```diff
@@ -200,33 +204,33 @@ paths:
...
-                              description: The price for a product including estimates for tax
+                                  description: The estimated tax inclusive price for this product based on the provided customer group.
+                              description: The price for a product including estimates for tax.
                             maximum:
                               type: object
                               properties:
                                 as_entered:
                                   type: number
-                                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:223]:
```suggestion
                                  description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.
```

File: reference/pricing.sf.yml [hunk 8]
```diff
@@ -236,33 +240,33 @@ paths:
...
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:259]:
```suggestion
                                  description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 9]
```diff
@@ -387,78 +391,78 @@ components:
...
-                description: The (optional) RRP/retail price configured for this product
+                    description: The estimated tax inclusive price for this product based on the provided customer group.
+                description: The (optional) RRP/retail price configured for this product.
               sale_price:
                 type: object
                 properties:
                   as_entered:
                     type: number
-                    description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                    description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:407]:
```suggestion
                    description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 10]
```diff
@@ -387,78 +391,78 @@ components:
...
-                description: The price for a product including estimates for tax
+                    description: The estimated tax inclusive price for this product based on the provided customer group.
+                description: The price for a product including estimates for tax.
               minimum_advertised_price:
                 type: object
                 properties:
                   as_entered:
                     type: number
-                    description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                    description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:423]:
```suggestion
                    description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.
```

File: reference/pricing.sf.yml [hunk 11]
```diff
@@ -387,78 +391,78 @@ components:
...
-                description: The price for a product including estimates for tax
+                    description: The estimated tax inclusive price for this product based on the provided customer group.
+                description: The price for a product including estimates for tax.
               calculated_price:
                 type: object
                 properties:
                   as_entered:
                     type: number
-                    description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                    description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:455]:
```suggestion
                    description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.
```

File: reference/pricing.sf.yml [hunk 12]
```diff
@@ -467,34 +471,34 @@ components:
                     properties:
                       as_entered:
                         type: number
-                        description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                        description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:474]:
```suggestion
                        description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.
```

File: reference/pricing.sf.yml [hunk 13]
```diff
@@ -503,45 +507,45 @@ components:
...
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:510]:
```suggestion
                        description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 14]
```diff
@@ -564,70 +586,70 @@ components:
           properties:
             as_entered:
               type: number
-              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:589]:
```suggestion
              description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 15]
```diff
@@ -564,70 +586,70 @@ components:
...
-          description: The price for a product including estimates for tax
+              description: The estimated tax inclusive price for this product based on the provided customer group.
+          description: The price for a product including estimates for tax.
         maximum:
           type: object
           properties:
             as_entered:
               type: number
-              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:605]:
```suggestion
              description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 16]
```diff
@@ -564,70 +586,70 @@ components:
...
+          description: The price for a product including estimates for tax.
+      description: For estimated prices, the minimum/maximum price that will typically apply to this product.
       x-internal: false
     TaxPrice:
       type: object
       properties:
         as_entered:
           type: number
-          description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+          description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:623]:
```suggestion
          description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 17]
```diff
@@ -564,70 +586,70 @@ components:
...
-          description: The product ID this price was generated for
+          description: The (required) product ID of the item.
         variant_id:
           type: integer
-          description: The (optional) variant ID this price was generated for
+          description: The (optional) variant ID of the item.
         options:
           type: array
-          description: The optional product option configuration this price was generated for
+          description: The optional product option configuration this price was generated for.
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:646]:
```suggestion
          description: The optional product option configuration for which this price was generated.

```

File: reference/pricing.sf.yml [hunk 18]
```diff
@@ -636,81 +658,81 @@ components:
...
-          description: The price for a product including estimates for tax
+              description: The estimated tax inclusive price for this product based on the provided customer group.
+          description: The price for a product including estimates for tax.
         calculated_price:
           type: object
           properties:
             as_entered:
               type: number
-              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+              description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:725]:
```suggestion
              description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 19]
```diff
@@ -719,34 +741,34 @@ components:
               properties:
                 as_entered:
                   type: number
-                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:744]:
```suggestion
                  description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 20]
```diff
@@ -719,34 +741,34 @@ components:
...
-              description: The price for a product including estimates for tax
+                  description: The estimated tax inclusive price for this product based on the provided customer group.
+              description: The price for a product including estimates for tax.
             maximum:
               type: object
               properties:
                 as_entered:
                   type: number
-                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:760]:
```suggestion
                  description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 21]
```diff
@@ -755,34 +777,34 @@ components:
               properties:
                 as_entered:
                   type: number
-                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:780]:
```suggestion
                  description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 22]
```diff
@@ -755,34 +777,34 @@ components:
...
-              description: The price for a product including estimates for tax
+                  description: The estimated tax inclusive price for this product based on the provided customer group.
+              description: The price for a product including estimates for tax.
             maximum:
               type: object
               properties:
                 as_entered:
                   type: number
-                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex)
+                  description: The price provided by the merchant as entered in their catalog/price list (may be inc or ex).
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:796]:
```suggestion
                  description: The price provided by the merchant as entered in their catalog/price list. May include or exclude tax.

```

File: reference/pricing.sf.yml [hunk 23]
```diff
@@ -805,38 +827,42 @@ components:
...
+          description: The currency of prices to be returned for this request.
           example: USD
         customer_group_id:
           type: integer
-          description: 'The customer group relevant for any customer group pricing, tax values, etc.'
+          description: The customer group relevant for any customer group pricing, tax values, etc.
         items:
           type: array
-          description: The items to fetch prices for
+          description: The items to fetch prices for.
```
Comment (bc-andreadao) [inline reference/pricing.sf.yml:856]:
```suggestion
          description: The items for which to fetch prices.

```

**Review by bc-andreadao — COMMENTED** [review]
I left some copyedits to take or leave

File: reference/pricing.sf.yml (hunk 16 above)
Comment (bc-tgomez) [reply reference/pricing.sf.yml]: Thanks a mil for these edits!

**Pull request merged**